src/features/wallet/context/EvmWalletContext.tsx

```diff
@@ -15,12 +22,106 @@ import {
 } from '@rainbow-me/rainbowkit/wallets';
 import { PropsWithChildren, useMemo } from 'react';
 import { createClient, fallback, http } from 'viem';
-import { WagmiProvider, createConfig } from 'wagmi';
+import { WagmiProvider, createConfig, createConnector } from 'wagmi';
+import { injected } from 'wagmi/connectors';
 import { APP_NAME } from '../../../consts/app';
 import { config } from '../../../consts/config';
 import { Color } from '../../../styles/Color';
 import { useMultiProvider } from '../../chains/hooks';
 
+export const universalProfilesWallet = ({ projectId }: { projectId: string }): Wallet => {
```

Comment:
Suggest putting this in a separate file and import it in this file (to keep the file minimal like it is currently, 72 lines of code)

https://github.com/lukso-network/service-bridge-hyperlane-warp-website-original/blob/main/src/features/wallet/context/EvmWalletContext.tsx